controllers/dspipeline_params_test.go

```diff
@@ -258,3 +260,26 @@ func TestExtractParams_CABundle(t *testing.T) {
 func strPtr(v string) *string {
 	return &v
 }
+
+func TestExtractParams_WithCustomKfpLauncherConfigMap(t *testing.T) {
+	ctx, params, client := CreateNewTestObjects()
+	cmDataExpected := map[string]string{
+		"this-is-the-only-thing": "that-should-be-in-kfp-launcher-now",
+	}
+	cm := v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "my-custom-kfp-launcher",
+			Namespace: "testnamespace",
+		},
+		Data: cmDataExpected,
+	}
+	err := client.Create(ctx, &cm)
+	assert.Nil(t, err)
```

Comment:
nit: `require` stops the execution when the condition fails. Continuing the execution (by using `assert`) may pollute the logs and make it harder to find the reason why the test failed.
```suggestion
	require.Nil(t, err)  // require needs to be imported
```